model_api/python/openvino/model_api/models/classification.py

```diff
@@ -60,14 +60,21 @@ def __init__(self, inference_adapter, configuration=dict(), preload=False):
                 self.load()
             return
 
-        addOrFindSoftmaxAndTopkOutputs(
-            self.inference_adapter, self.topk, self.output_raw_scores
-        )
+        try:
+            addOrFindSoftmaxAndTopkOutputs(
+                self.inference_adapter, self.topk, self.output_raw_scores
+            )
+            self.embedded_topk = True
+            self.out_layer_names = ["indices", "scores"]
+            if self.output_raw_scores:
+                self.out_layer_names.append(self.raw_scores_name)
+        except (RuntimeError, AttributeError):
+            self.embedded_topk = False
+            self.out_layer_names = _get_non_xai_names(self.outputs.keys())
+            self.raw_scores_name = self.out_layer_names[0]
```

Comment:
Just a note, if you are ever going to modify `classification.py` further, it may be simpler to remove ngraph based postprocessing from C++ and Python

Reply:
Ok, let's see if this fallback logic requires any modification in the future.